recipes/bw_graph_tools/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "bw_graph_tools" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bw_graph_tools-{{ version }}.tar.gz
+  sha256: 100893a2a802f284f0bc8f6ac9636a446b76f5ba518609acd8e9f7bbcbb4f323
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=68.0
+    - pip
+  run:
+    - python >=3.9
+    - bw2calc >=2.0.dev17
+    - bw_processing >=0.9.5
+    - matrix_utils >=0.4.3
+    - numpy <2
+    - scipy
+
+test:
+  imports:
+    - bw_graph_tools
+  commands:
+    - pytest
```

Comment:
You need to include the source files like:
https://github.com/conda-forge/pillow-feedstock/blob/main/recipe/meta.yaml#L72

If you want `pip check` here to pass successfully, I would ensure that all your dependencies have it too. Make PRs to add it to the packages you care about and see if any of them fail.